src/js/tabs/osd.js

```diff
@@ -688,11 +682,7 @@ OSD.loadDisplayFields = function() {
             text: 'osdTextElementHorizonSidebars',
             desc: 'osdDescElementHorizonSidebars',
             defaultPosition() {
-                let position = 194;
-                if (OSD.constants.VIDEO_TYPES[OSD.data.video_system] !== 'NTSC') {
-                    position += FONT.constants.SIZES.LINE;
-                }
-                return position;
+                let position = (OSD.constants.VIDEO_COLS[videoType] >> 1) + ((OSD.constants.VIDEO_ROWS[videoType] >> 1) - 2) * OSD.constants.VIDEO_COLS[videoType] - 1;
```

Comment:
Probably should be
```suggestion
                return (OSD.constants.VIDEO_COLS[videoType] >> 1) + ((OSD.constants.VIDEO_ROWS[videoType] >> 1) - 2) * OSD.constants.VIDEO_COLS[videoType] - 1;
```
or define on line 691 and return position.

Reply:
Agreed. There were three such instances left over from a code simplification. Fixed.